### DISCUSSION TITLE
After the installation is complete, opening the webpage displays a blank white screen.

### DISCUSSION THREAD

**Discussion by qjmltz** [Sanster/IOPaint#506] [Q&A]
<img width="876" alt="3b931dc181bad166eb5be2bfd7eadef" src="https://github.com/Sanster/IOPaint/assets/88932616/9df88073-d610-4e68-a280-f3b30421e979">

**Comment by zzzzzzzs** [comment]
I had the same problem

**Comment by Sanster** [comment]
Please refer to the method in this issue: https://github.com/Sanster/IOPaint/issues/460
